EIPS/eip-7007.md

```diff
@@ -0,0 +1,160 @@
+---
+eip: 7007
+title: zkML AIGC-NFTs
+description: An ERC-721 extension interface for zkML based AIGC-NFTs.
+author: Cathie So (@socathie), Xiaohang Yu <xhyumiracle@hyperoracle.io>, Huaizhe Xu (@HuaizheXu), Kartin <kartin@hyperoracle.io>
+discussions-to: https://ethereum-magicians.org/t/eip-7007-zkml-aigc-nfts-an-erc-721-extension-interface-for-zkml-based-aigc-nfts/14216
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-05-10
+requires: 165, 721
+---
+
+## Abstract
+
+The zkML AIGC-NFTs standard is an extension of the [ERC-721](./eip-721.md) token standard for AI Generated Content. It proposes a set of interfaces for basic interactions and enumerable interactions for AIGC-NFTs. The standard includes a new mint event and a JSON schema for AIGC-NFT metadata. Additionally, it incorporates zkML capabilities to enable verification of AIGC-NFT ownership. In this standard, the `tokenId` is indexed by the `prompt`.
+
+## Specification
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+**Every compliant contract must implement the [ERC-7007](./eip-7007.md), [ERC-721](./eip-721.md), and [ERC-165](./eip-165.md) interfaces.**
+
+The zkML AIGC-NFTs standard includes the following interfaces: 
+
+ERC-7007: Defines a mint event and a mint function for minting AIGC-NFTs. It also includes a verify function to check the validity of a combination of prompt and proof using zkML techniques.
```

Comment:
I'm assuming the bot made you put a dash here? If you use backticks to indicate inline code, the bot won't complain:

```suggestion
`IERC7007`: Defines a mint event and a mint function for minting AIGC-NFTs. It also includes a verify function to check the validity of a combination of prompt and proof using zkML techniques.
```